library/alloc/src/string.rs

```diff
@@ -3094,6 +3100,91 @@ impl fmt::Write for String {
     }
 }
 
+/// Placeholder docs.
+#[unstable(feature = "into_chars", reason = "new API", issue = "none")]
+pub struct IntoChars {
+    bytes: Vec<u8>,
```

Comment:
Since we have [vec::IntoIter::advance_back_by](https://doc.rust-lang.org/std/iter/trait.DoubleEndedIterator.html#method.advance_back_by), I will convert the code to use `vec::IntoIter`.